packages/openapi-request-validator/test/data-driven/accept-mutipart-with-boundary.js

```diff
@@ -7,6 +7,7 @@ module.exports = {
         'multipart/form-data': {
           schema: {
             type: 'object',
+            required: ['files'],
```

Comment:
@mbsimonovic does this reproduce the bug?

Reply:
yes but the request has to be modified as well: `files: ['binary file 1']` will satisfy this required clause so it does validate. i didn't want for the build to fail on this pr, so i've made it pass by setting files to an array of strings. If multer is used, then this won't be a list of strings, but list of objects, so the test will fail just like in the example by @mendelk